packages/react-core/src/components/Tabs/TabTitleIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Tabs/tabs';
+
+export interface TabTitleIconProps extends React.HTMLProps<HTMLSpanElement> {
+  /** Icon to be rendered inside the tab button title. */
+  children?: React.ReactNode;
+  /** additional classes added to the tab title icon */
+  className?: string;
+}
+
+export const TabTitleIcon: React.FunctionComponent<TabTitleIconProps> = ({
+  children,
+  className = '',
```

Comment:
Nit: we don't have to set these to a default of empty string anymore on `v4`.

Reply:
I followed the API we have in the drawer component for width.